### PR TITLE
Failed Uphold tip still shows success screen in tipping banner

### PR DESCRIPTION
**Pull request by sujitacharya2005**
Resolves https://github.com/brave/brave-browser/issues/26815

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

1. Pending case it will show pending status , e.g. unverified site case if send tip will pending.
2. If brave verified and custodian not verified then it will show failure
3. otherwise it will success
4. for monthly tip always show success.

|  | |  | |
| ------------- | ------------- |------------- | ------------- |
| image |  image |   image | image |

### CODE REVIEW

File: android/java/res/values/brave_colors.xml [hunk 1]
```diff
@@ -227,6 +227,12 @@
     <color name="rewards_tipping_panel_divider_color">#80AEB1C2</color>
     <color name="rewards_tipping_black_color">#000000</color>
     <color name="rewards_tipping_white_color">#FFFFFF</color>
+    <color name="rewards_tipping_publisher_name_color">#F8F9FA</color>
+    <color name="rewards_tipping_radio_group_bg">#E9E9F4</color>
+    <color name="rewards_tipping_text_color">#212529</color>
+
```
Comment (deeppandya) [inline android/java/res/values/brave_colors.xml:233]: Can we remove extra new lines here ?

File: android/java/res/drawable/rewards_tipping_radio_group_bg.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+	<solid android:color="@color/rewards_tipping_radio_group_bg"/>
+    <corners android:radius="20dp"/>
+</shape>
```
Comment (deeppandya) [inline android/java/res/drawable/rewards_tipping_radio_group_bg.xml:5]: Add new line at the end as per chromium guidelines

File: android/java/res/drawable/rewards_tipping_radio_button_selector.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_checked="false" android:drawable="@drawable/rewards_tipping_radio_button_normal" />
+    <item android:state_checked="true" android:drawable="@drawable/rewards_tipping_radio_button_selected" />
+</selector>
```
Comment (deeppandya) [inline android/java/res/drawable/rewards_tipping_radio_button_selector.xml:5]: Add new line at the end as per chromium guidelines

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsTipConfirmationFragment.java [hunk 4]
```diff
@@ -98,7 +98,10 @@ private void initView(
         mNextTipDateText = view.findViewById(R.id.next_tip_date_text);
         mNextTipDateValue = view.findViewById(R.id.next_tip_date_value);
 
-        if (monthlyContribution) {
+        if (status == BraveRewardsSiteBannerActivity.TIP_PENDING) {
+            tipTypeDescriptionTextView.setText(R.string.pending_tip_text);
+            oneTtimeTipNote.setVisibility(View.INVISIBLE);
```
Comment (deeppandya) [inline android/java/org/chromium/chrome/browser/rewards/BraveRewardsTipConfirmationFragment.java:103]: Should it be View.GONE?

Comment (sujitacharya2005) [reply android/java/org/chromium/chrome/browser/rewards/BraveRewardsTipConfirmationFragment.java:103]: In success case it's visible and for pending case invisible.
That space should be occupied.

File: android/java/res/values/brave_colors.xml (hunk 1 above)
Comment (sujitacharya2005) [reply android/java/res/values/brave_colors.xml:233]: done

File: android/java/res/drawable/rewards_tipping_radio_group_bg.xml (hunk 2 above)
Comment (sujitacharya2005) [reply android/java/res/drawable/rewards_tipping_radio_group_bg.xml]: done

File: android/java/res/drawable/rewards_tipping_radio_button_selector.xml (hunk 3 above)
Comment (sujitacharya2005) [reply android/java/res/drawable/rewards_tipping_radio_button_selector.xml]: done

**Review by deeppandya — APPROVED** [review]
LGTM

**Comment by GeetaSarvadnya** [comment]
Verification is inprogress on `Vivo X70 Pro version 12` running `Bravemonoarm64.apk_1.48.9`

- Verified the test plan from https://github.com/brave/brave-core/pull/16057#issue-1460413599

**Pull request merged**